python/ray/experimental/compiled_dag_ref.py

```diff
@@ -1,10 +1,17 @@
 import asyncio
+import logging
 from typing import Any, List, Optional
 
 import ray
+from ray._private.ray_constants import env_bool
 from ray.exceptions import RayTaskError
 from ray.util.annotations import PublicAPI
 
+logger = logging.getLogger(__name__)
+
+RAY_IGNORE_UNHANDLED_ERRORS_ENV_VAR = "RAY_IGNORE_UNHANDLED_ERRORS"
+RAY_IGNORE_UNHANDLED_ERRORS = env_bool(RAY_IGNORE_UNHANDLED_ERRORS_ENV_VAR, False)
```

Comment:
these should be prefixed with `RAY_ADAG` as they are only relevant to adag

Reply:
actually this is the same env var as what we have in Ray itself https://github.com/ray-project/ray/blob/ee320aa6670514b85e77fce8d6903affcf883cc4/python/ray/_private/worker.py#L799

imo it is better unifying. I will use the common constant for the env var instead

Reply:
Got it. Can you add a comment here: people tend to change this to make names consistent if they are not aware of the reuse.